integrations/jira/.port/resources/blueprints.json

```diff
@@ -121,7 +111,70 @@
         "title": "Subtasks",
         "required": false,
         "many": true
+      },
+      "assignee": {
+        "target": "jiraUser",
+        "title": "Assignee",
+        "required": false,
+        "many": false
+      },
+      "reporter": {
+        "target": "jiraUser",
+        "title": "Reporter",
+        "required": false,
+        "many": false
       }
     }
+  },
+  {
+    "identifier": "jiraUser",
+    "title": "Jira User",
+    "icon": "User",
+    "description": "A Jira user account",
+    "schema": {
+      "properties": {
+        "emailAddress": {
+          "title": "Email",
+          "type": "string",
+          "format": "email",
+          "description": "User's email address"
+        },
+        "displayName": {
```

Comment:
make sure to change order of blueprints, as the jira user needs to be created prior to the jira issue so that the jira issue will relate to it